src/app.js

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
 const express = require('express');
 const app = express();
 const cors = require('cors')
```

Comment:
[Functional Error]
You need to reference your user/auth subrouter here in the app.js for it to work.
As it stands this wouldn't route so it wouldn't work if you tried to hook up postman.

Reply:
Referenced the:
app.use('/api/v1/user', userRouter);
in app.js
I hope I understood you correctly regarding 'user' instead of 'auth'. Is is ok that I also changed the path with /user instead of '/auth' ? or I misunderstood you?

Reply:
I also added the code from ticket Pract-31 regarding adding the subrouter (in server.js). And closed the PR Pract-31